cumulus/parachains/integration-tests/emulated/common/src/lib.rs

```diff
@@ -164,3 +159,13 @@ pub mod validators {
 		)]
 	}
 }
+
+pub mod snowbridge {
+	use hex_literal::hex;
+	// Address of WETH ERC20 token contract on remote Ethereum network
+	pub const WETH: [u8; 20] = hex!("fff9976782d46cc05630d1f6ebab18b2324d6b14");
+	// The Ethereum network chain ID. In this case, Sepolia testnet's chain ID.
+	pub const CHAIN_ID: u64 = 11155111;
```

Comment:
why not be explicit?

```suggestion
	pub const SEPOLIA_ID: u64 = 11155111;
```

Reply:
Addressed in https://github.com/paritytech/polkadot-sdk/pull/8014/commits/f6cc5accfb8efede06dcab6ba80299fa6a30fffc.